src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -52,7 +52,17 @@ public override bool IsServerDelegatable(CallNode callNode, TexlBinding binding)
             Contracts.AssertValue(binding);
             Contracts.Assert(binding.GetInfo(callNode).Function is UserDefinedFunction udf && udf.Binding != null);
 
-            return base.IsServerDelegatable(callNode, binding) || IsDelegatable;
+            if (base.IsServerDelegatable(callNode, binding) || IsDelegatable)

```

Comment:
nit: can we just have 1 return like 
```suggestion
   return a || b || c
```